src/pages/HomePage/pages/LifeCenter/components/LifeCenterCard.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { LifeCenterType } from "../LifeCenter";
+import Action from "@/components/Action";
+import ellipse from "@/assets/ellipse.svg";
+import { Dialog } from "@/components/Dialog";
+import { encodeQuery, showDeleteDialog } from "@/pages/HomePage/utils";
+import { api } from "@/utils";
+import { useDelete } from "@/CustomHooks/useDelete";
+import { useNavigate } from "react-router-dom";
+
+interface Props {
+  item: LifeCenterType;
+  handleEdit: (item: LifeCenterType) => void;
+  handleDelete: (id: string) => void;
+}
+
+const InfoRow = ({ icon, label }: { icon: string; label: React.ReactNode }) => (
+  <div className="flex items-center gap-1">
+    <img src={icon} className="size-5" />
+    <p>{label}</p>
+  </div>
+);
+
+const LifeCenterCard: React.FC<Props> = ({ item, handleEdit, handleDelete }) => {
```

Comment:
no default export 
also since this is the main exported file please move the infoRow to the bottom